.github/workflows/test.yml

```
@@ -19,6 +19,11 @@ jobs:
         uses: olafurpg/setup-scala@v10
       - name: Cache
         uses: coursier/cache-action@v5
+      - name: Setup Mill
+        uses: jodersky/setup-mill@master
```

It would be better to pin this to a version
```suggestion
      - name: Setup Mill
        uses: jodersky/setup-mill@v0.2.3
```